packages/cra-template-typescript/template/src/services/sagas/index.ts

```diff
@@ -1,11 +1,15 @@
 import { all } from 'redux-saga/effects';
 
+import { saga as serviceWorker } from 'modules/service-worker';
+import { saga as network } from 'modules/network';
 // @use-auth-module-begin
 import { saga as auth } from 'modules/auth';
 // @use-auth-module-end
 
 export default function* appSaga() {
     yield all([
+        serviceWorker(),
+        network(),
```

Comment:
I might have missed it, but shouldn't these sagas run conditionaly only if `useServiceWorker` option is true?

Reply:
No, because then could happen that SW that has been already registered, couldn't be ever unregistered.